### ISSUE TITLE
Issue installing package with `devtools` - installed from source instead 

### ISSUE THREAD

**Issue by terencewtli** [sshen82/BandNorm#3]
Hi,

Thank you for this package! I tried installing this package with the following command:

`devtools::install_github('sshen82/BandNorm', build_vignettes = TRUE)`

I got the error below:

``` preparing ‘BandNorm’: (813ms)
 checking DESCRIPTION meta-information ... 
 installing the package to build vignettes
 creating vignettes (38.8s)
 --- re-building ‘BandNorm-tutorial.Rmd’ using rmarkdown
 Warning in engine$weave(file, quiet = quiet, encoding = enc) :
   Pandoc (>= 1.12.3) not available. Falling back to R Markdown v1. 
 Error: processing vignette 'BandNorm-tutorial.Rmd' failed with diagnostics:
 The 'markdown' package should be installed and declared as a dependency of the 'BandNorm' package (e.g.
, in the 'Suggests' field of DESCRIPTION), because the latter contains vignette(s) built with the 'mar
own' package. Please see https://github.com/yihui/knitr/issues/1864 for more information.
 --- failed re-building ‘BandNorm-tutorial.Rmd’

 --- re-building ‘scGAD-tutorial.Rmd’ using rmarkdown
 Warning in engine$weave(file, quiet = quiet, encoding = enc) :
   Pandoc (>= 1.12.3) not available. Falling back to R Markdown v1. 
 Error: processing vignette 'scGAD-tutorial.Rmd' failed with diagnostics:
 The 'markdown' package should be installed and declared as a dependency of the 'BandNorm' package (e.g.
, in the 'Suggests' field of DESCRIPTION), because the latter contains vignette(s) built with the 'mar
own' package. Please see https://github.com/yihui/knitr/issues/1864 for more information.
 --- failed re-building ‘scGAD-tutorial.Rmd’

 SUMMARY: processing the following files failed:
   ‘BandNorm-tutorial.Rmd’ ‘scGAD-tutorial.Rmd’

 Error: Vignette re-building failed.
 Execution halted
Error: Failed to install 'BandNorm' from GitHub:
System command 'R' failed, exit status: 1, stdout + stderr (last 10 lines):
   Pandoc (>= 1.12.3) not available. Falling back to R Markdown v1. 
 Error: processing vignette 'scGAD-tutorial.Rmd' failed with diagnostics:
 The 'markdown' package should be installed and declared as a dependency of the 'BandNorm' package (e.g.
, in the 'Suggests' field of DESCRIPTION), because the latter contains vignette(s) built with the 'mar
own' package. Please see https://github.com/yihui/knitr/issues/1864 for more information.
 --- failed re-building ‘scGAD-tutorial.Rmd’

 SUMMARY: processing the following files failed:
   ‘BandNorm-tutorial.Rmd’ ‘scGAD-tutorial.Rmd’

 Error: Vignette re-building failed.
```

I have the `rmarkdown` and `markdown` packages installed, but the installation still didn't work. I checked out the Github [issue](https://github.com/yihui/knitr/issues/1864 ) and as suggested, edited the `DESCRIPTION` file to include `markdown` in the `Suggests` tab, then installed from source successfully:

`install.packages('/path/to/BandNorm', repos = NULL, source = TRUE)`

To summarize, I was wondering if I overlooked a dependency when trying to install using `devtools`, and whether or not `markdown` should ultimately be added to the `DESCRIPTION` file. Thank you!
